src/service/app/alert.ts

```diff
@@ -1,20 +1,11 @@
 import { fetchMaintenanceInfo } from '../http/fetcher'
-import { AppDispatch } from '../../contexts/reducer'
-import { AppActions } from '../../contexts/actions'
+import { setMaintenanceMsg } from '../../components/Alert'
 
-export const getMaintenanceInfo = (dispatch: AppDispatch) => {
+export const getMaintenanceInfo = () => {
```

Comment:
This API can be removed now because maintenance status is toggled by frontend runtime variables

Reply:
removed